var/spack/repos/builtin/packages/espresso/package.py

```diff
@@ -91,38 +94,62 @@ def install(self, spec, prefix):
             options.append('--enable-openmp')
 
         if '+scalapack' in spec:
-            options.append('--with-scalapack=yes')
+            scalapack_option = 'intel' if '%intel' in spec else 'yes'
```

Comment:
similar here, what is `intel` here? I guess what they mean is when one use `mkl`'s `scalapack`. Otherwise one may still be using `netlib-scalapack` compiled with `intel` compiler. Please change this accordingly.